inference-engine/tests/functional/plugin/shared/include/behavior/preprocessing.hpp

```diff
@@ -110,9 +110,12 @@ struct PreprocessingPrecisionConvertTest :
     }
 
     void Validate() override {
+        // w/a: copy of original function is required to provide correct op coverage report (overflow of convert counter issue)
```

Comment:
Extra changes. Please remove the changes from the pr

Reply:
Fixed